spacy_stanfordnlp/language.py

```diff
@@ -22,6 +25,8 @@ def __init__(self, snlp, meta=None, **kwargs):
         RETURNS (spacy.language.Language): The nlp object.
         """
         lang = snlp.config["lang"]
+        self.snlp = snlp
+        self.svecs = snlp.processors['pos'].pretrain
```

Comment:
What happens here if the user passes in a StanfordNLP pipeline that doesn't have the `pos` processor enabled?